src/test/kotlin/com/atlassian/performance/tools/hardware/IntegrationTestRuntime.kt

```diff
@@ -16,20 +13,28 @@ import java.util.*
 object IntegrationTestRuntime {
 
     val rootWorkspace = RootWorkspace(Paths.get("build"))
+    private const val roleArn: String = "arn:aws:iam::695067801333:role/server-gdn-bamboo"
+    private val webIdentityTokeFile: String = System.getenv("AWS_WEB_IDENTITY_TOKEN_FILE")
+    private val region = Regions.EU_WEST_1
 
     fun prepareAws() = Aws(
         credentialsProvider = AWSCredentialsProviderChain(
             STSAssumeRoleSessionCredentialsProvider.Builder(
-                "arn:aws:iam::695067801333:role/server-gdn-bamboo",
+                roleArn,
                 UUID.randomUUID().toString()
             ).build(),
             ProfileCredentialsProvider("jpt-dev"),
             EC2ContainerCredentialsProviderWrapper(),
+            WebIdentityTokenCredentialsProvider.builder()
+                .webIdentityTokenFile(webIdentityTokeFile)
+                .roleArn(roleArn)
+                .roleSessionName(UUID.randomUUID().toString())
+                .build(),
             DefaultAWSCredentialsProviderChain()
         ),
-        region = Regions.EU_WEST_1,
-        regionsWithHousekeeping = listOf(Regions.EU_WEST_1),
-        capacity = TextCapacityMediator(Regions.EU_WEST_1),
+        region = region,
+        regionsWithHousekeeping = listOf(region),
```

Comment:
This region is "accidentally" the same. The `regionsWithHousekeeping` list should always be literal, no reuse.
```suggestion
        regionsWithHousekeeping = listOf(Regions.EU_WEST_1),
```